crates/node-bindings/src/parcel/parcel/tracing_setup.rs

```diff
@@ -1,73 +0,0 @@
-use anyhow::anyhow;
-use napi_derive::napi;
-
-/// JavaScript provided options to configure the `tracing_subscriber` rust logs into a file or the
-/// console.
-#[napi(object)]
-pub struct ParcelTracingOptions {
-  /// Enable tracing
-  pub enabled: bool,
-  /// If set to some, will trace to a file with the given options, otherwise the console will be
-  /// used.
-  pub output_file_options: Option<ParcelTracingOutputFileOptions>,
-}
-
-impl Default for ParcelTracingOptions {
-  fn default() -> Self {
-    Self {
-      enabled: true,
-      output_file_options: Some(ParcelTracingOutputFileOptions {
-        directory: std::env::temp_dir().to_string_lossy().to_string(),
-        prefix: "parcel-tracing".to_string(),
-        max_files: 4,
-      }),
-    }
-  }
-}
-
-/// Output file configuration.
-/// Tracing log files will be rotated hourly on the provided directory.
-#[napi(object)]
-pub struct ParcelTracingOutputFileOptions {
-  /// The directory where the log files will be written.
-  pub directory: String,
-  /// A prefix for the log file names.
-  pub prefix: String,
-  /// The maximum number of rotated files to keep.
-  pub max_files: u32,
-}
-
-/// This is a guard, when it's dropped certain resources related to tracing might flush to disk or
-/// be released. It should be kept alive for the duration of the program.
-pub struct ParcelTracingGuard {
-  guard: Option<tracing_appender::non_blocking::WorkerGuard>,
-}
-
-/// Set-up tracing based on JavaScript provided options and return a guard.
-pub fn setup_tracing(options: &Option<ParcelTracingOptions>) -> anyhow::Result<ParcelTracingGuard> {
```

Comment:
This already will only be initialized once because it'll fail if you initialise twice, we can just ignore the failure

Reply:
Updated to ignore the failure